app/presenters/advanced_search_finder_presenter.rb

```diff
@@ -23,6 +23,12 @@ def taxon_link
     link_to taxon['title'], taxon['base_path'], class: 'taxon-link'
   end
 
+  def breadcrumbs
+    @data ||= GovukNavigationHelpers::TaxonBreadcrumbs.new(content_item).breadcrumbs
+    filtered = @data[:breadcrumbs].reject { |bc| bc[:is_page_parent] || bc[:is_current_page] }
```

Comment:
govuk_navigation helpers 9.2.1 ensures that current page breadcrumbs are no longer returned (see https://github.com/alphagov/govuk_navigation_helpers/pull/125/files) so once the version in this app is bumped we should no longer need to check for `:is_current_page` 🎉